src/mainwindow.cpp

```diff
@@ -380,6 +398,14 @@ void MainWindow::aboutKDAB()
     dialog.exec();
 }
 
+void MainWindow::openSettingsDialog() {
+    m_settingsDialog->setWindowTitle(tr("Paths and Architecture Settings"));
+    m_settingsDialog->setWindowIcon(QPixmap(QStringLiteral(":/images/kdablogo.png")));
```

Comment:
use QIcon instead of QPixmap

Reply:
also, don't use the kdab logo but instead the hotspot logo, maybe just inherit the current window icon?

```
mSettingsDialog->setWindowIcon(windowIcon());
```